### PR TITLE
release: 1.4.1

### PR DESCRIPTION
**Pull request by ematipico**
## Summary

Release 1.4.1 with some bug fix

## Test Plan

Proof read the changelog

### CODE REVIEW

**Comment by ematipico** [comment]
@faultyserver Will you be able to add [your fixes](https://github.com/biomejs/biome/pulls?q=is%3Apr+sort%3Aupdated-desc+author%3Afaultyserver+is%3Amerged) in the changelog? 

Here's how: https://github.com/biomejs/biome/blob/main/CONTRIBUTING.md#writing-a-changelog-line

**Comment by Conaclos** [comment]
This coudl be great to document the #928 fix in thec hangelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,24 +8,28 @@ Read our [guidelines to categorize a change](https://biomejs.dev/internals/versi
...
 Read our [guidelines for writing a good changelog entry](https://github.com/biomejs/biome/blob/main/CONTRIBUTING.md#changelog).
 
-## Unreleased
-
-### Analyzer
-
-### CLI
-
-### Configuration
+## 1.4.1 (2023-29-11)
```
Comment (SuperchupuDev) [inline CHANGELOG.md:11]: we follow `yyyy-mm-dd`, not `yyyy-dd-mm`

Comment (ematipico) [reply CHANGELOG.md:11]: Can you add a suggestion and commit please?

Comment (ematipico) [reply CHANGELOG.md:11]: Tomorrow then :)

Comment (SuperchupuDev) [inline CHANGELOG.md:11]:
```suggestion
## 1.4.1 (2023-11-30)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,24 +8,28 @@ Read our [guidelines to categorize a change](https://biomejs.dev/internals/versi
...
+
+- Match Prettier's breaking strategy for `ArrowChain` layouts [#934](https://github.com/biomejs/biome/pull/934). Contributed by @faultyserver
+
+- Fix double-printing of leading comments in arrow chain expressions [#951](https://github.com/biomejs/biome/pull/951). Contributed by @faultyserver
 
 ### Linter
 
+#### Bug fixes
+
+- Fix [#910](https://github.com/biomejs/biome/issues/910), where the rule `noSvgWithoutTitle` should skip elements that have `aria-hidden` attributes.
```
Comment (unvalley) [inline CHANGELOG.md:31]:
```suggestion
- Fix [#910](https://github.com/biomejs/biome/issues/910), where the rule `noSvgWithoutTitle` should skip elements that have `aria-hidden` attributes. Contributed by @vasucp1207
```

File: website/src/content/docs/internals/changelog.mdx [hunk 3]
```diff
@@ -14,24 +14,28 @@ Read our [guidelines to categorize a change](https://biomejs.dev/internals/versi
...
+
+- Match Prettier's breaking strategy for `ArrowChain` layouts [#934](https://github.com/biomejs/biome/pull/934). Contributed by @faultyserver
+
+- Fix double-printing of leading comments in arrow chain expressions [#951](https://github.com/biomejs/biome/pull/951). Contributed by @faultyserver
 
 ### Linter
 
+#### Bug fixes
+
+- Fix [#910](https://github.com/biomejs/biome/issues/910), where the rule `noSvgWithoutTitle` should skip elements that have `aria-hidden` attributes.
```
Comment (unvalley) [inline website/src/content/docs/internals/changelog.mdx:37]:
```suggestion
- Fix [#910](https://github.com/biomejs/biome/issues/910), where the rule `noSvgWithoutTitle` should skip elements that have `aria-hidden` attributes. Contributed by @vasucp1207
```

**Pull request merged**